gulpfile.js

```diff
@@ -24,12 +27,13 @@ let onError = function onError(err) {
   this.emit('end');
 };
 
-gulp.task('clean', () => {
+gulp.task('clobber-assets', () => {
```

Comment:
"clobbering" is a funny way of putting it but I suggest we keep the standard name "clean" so we can do `gulp clean`.

Reply:
Sure, there's a distinction in Rails between cleaning, removing old assets, and clobbering, removing all assets—but sure if that's uncommon I'll rename it